packages/office-ui-fabric-react/src/components/Dialog/examples/Dialog.LargeHeader.Example.tsx

```diff
@@ -2,65 +2,53 @@ import * as React from 'react';
 import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
 import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { ChoiceGroup } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { useBoolean } from '@uifabric/react-hooks';
 
-export interface IDialogLargeHeaderExampleState {
-  hideDialog: boolean;
-}
+const options = [
+  {
+    key: 'A',
+    text: 'Option A',
+  },
+  {
+    key: 'B',
+    text: 'Option B',
+    checked: true,
+  },
+  {
+    key: 'C',
+    text: 'Option C',
+    disabled: true,
+  },
+];
+const modelProps = {
+  isBlocking: false,
+  styles: { main: { maxWidth: 450 } },
+};
+const dialogContentProps = {
+  type: DialogType.largeHeader,
+  title: 'All emails together',
+  subText:
+    'Your Inbox has changed. ' + 'No longer does it include favorites, it is a singular destination for your emails.',
```

Comment:
Now that it fits in a single line it can probably be a single string without that `+` separation.